### PR TITLE
22853 update parent field

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/22853)

Last sprint we added a question to the 686c asking if the user was the Parent or Guardian of a child they were adding. After stakeholders had a chance to review this it turned out that we actually only care if the user is the Parent, not the guardian, and we should change the question to make that clear. To keep the `formData` consistent with the intent of the form this PR changes the name of the data from `isParentOrGuardian` to `isParent`

## Testing done
Unit tests have been updated

## Screenshots

Screen Shot 2021-04-08 at 12 26 43 PM

## Acceptance criteria
- [x] The question is in place on the proper page, as noted above
- [x] A PR has been requested for merging

### CODE REVIEW

**Review by micahchiang — APPROVED** [review]

File: src/applications/disability-benefits/686c-674/config/chapters/674/student-name-ssn/studentNameSSN.js [hunk 1]
```diff
@@ -55,8 +55,8 @@ export const uiSchema = {
           isChapterFieldRequired(formData, 'report674'),
       },
     },
-    isParentOrGuardian: {
-      'ui:title': 'Are you a parent or guardian of this child?',
+    isParent: {
+      'ui:title': "Are you this child's parent?",
```
Comment (Mottie) [inline src/applications/disability-benefits/686c-674/config/chapters/674/student-name-ssn/studentNameSSN.js:59]: wrong apostrophe.

**Review by Mottie — APPROVED** [review]

File: src/applications/disability-benefits/686c-674/config/chapters/674/student-name-ssn/studentNameSSN.js (hunk 1 above)
Comment (cohnjesse) [reply src/applications/disability-benefits/686c-674/config/chapters/674/student-name-ssn/studentNameSSN.js:59]: My understanding is that this apostrophe is only incorrect if the word following it is plural, which it is not.

Comment (Mottie) [reply src/applications/disability-benefits/686c-674/config/chapters/674/student-name-ssn/studentNameSSN.js:59]: ¯\_(ツ)_/¯

Comment (micahchiang) [reply src/applications/disability-benefits/686c-674/config/chapters/674/student-name-ssn/studentNameSSN.js:59]: I think it is the use of `'` vs `’`. It's sort of trivial, but the single quote is discouraged for use as an apostrophe. It isn't documented anywhere, but `’` is usually the proper thing to use for an apostrophe. Using `shift + option ]` types it out.

**Review by jhonny218 — APPROVED** [review]

**Pull request merged**